src/Directory.Build.props

```diff
@@ -0,0 +1,649 @@
+    CA1823 // Avoid unused private fields
+    CA1724 // Type names should not match namespaces
+    IDE0039 // Use local function
+    CS8620 // Argument cannot be used for parameter due to differences in the nullability of reference types.
+    CA1045 // Do not pass types by reference
+    CA1813 // Avoid unsealed attributes
+    -->
+    <!--
+    *********************************************************************
+    * Set the build output path, including the targeted framework name. *
+    *********************************************************************
+    -->
+    <PropertyGroup Label="Output">
+        <AppendTargetFrameworkToOutputPath>true</AppendTargetFrameworkToOutputPath>
+        <AppendRuntimeIdentifierToOutputPath>true</AppendRuntimeIdentifierToOutputPath>
+        <OutputPath>$(ProjectOutputPath)</OutputPath>
+        <BaseIntermediateOutputPath>$(ProjectObjPath)</BaseIntermediateOutputPath>
+    </PropertyGroup>
+    
+    <!--
+    ********************************************************************
+    * Set the absolute path for the documentation file of the project. *
+    * all documentation files will be in the 'Docs' folder.            *
+    ********************************************************************
+    -->
+    <PropertyGroup Label="Documentation">
+        <DocumentationFile>$(BuildFolder)Docs\$(ProjectName).xml</DocumentationFile>
+    </PropertyGroup>
+
+    <!--
+    ****************************************
+    * Setting for C# language and features *
+    ****************************************
+    -->
+    <PropertyGroup Label="C# Settings">
+        <LangVersion>preview</LangVersion>
+        <Features>strict</Features>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>disable</ImplicitUsings>
+    </PropertyGroup>
+
+    <!--
+    **********************
+    * Build optimization *
+    **********************
+    -->
+    <PropertyGroup Label="Build Settings">
+        <!-- see here for more info:
+             https://devblogs.microsoft.com/visualstudio/vs-toolbox-accelerate-your-builds-of-sdk-style-net-projects/ -->
+        <AccelerateBuildsInVisualStudio>true</AccelerateBuildsInVisualStudio>
+        
+        <!-- see here for more info:
+             https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#deterministic -->
+        <Deterministic>true</Deterministic>
+        
+        <!-- see here for more info:
+             https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#continuousintegrationbuild -->
+        <ContinuousIntegrationBuild>false</ContinuousIntegrationBuild>
+        <ContinuousIntegrationBuild Condition=" ( '$(GITHUB_ACTIONS)' == 'true' ) or
+                                                ( '$(TF_BUILD)' == 'true' ) ">true</ContinuousIntegrationBuild>
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
This second `true` without any condition looks suspicious, could it be a copy paste error or testing leftover?